python_modules/dagster/dagster_tests/definitions_tests/test_repository_definition.py

```diff
@@ -376,7 +376,10 @@ def needy(context):
     def bare():
         needy()
 
-    with pytest.raises(DagsterInvalidDefinitionError, match="Failed attempting to coerce Graph"):
+    with pytest.raises(
```

Comment:
these error messages change because the `Failed attempting to coerce Graph` is thrown in the except of a try/except guard around job creation - since the `DagsterInvalidDefinitionError` w/ missing resources only fires at repo init time, the Graph error isn't shown